app/src/main/java/com/kickstarter/services/ApiClient.java

```diff
@@ -248,11 +248,6 @@ public ApiClient(final @NonNull ApiService service, final @NonNull Gson gson) {
   }
 
   @Override
-  public @NonNull Observable<MessageThreadsEnvelope> fetchMessageThreads(final @NonNull Mailbox mailbox) {
```

Comment:
this was a weird temp method i made rather than just passing in a null project so we dont need it anymore / never really needed it oh well